inference-engine/thirdparty/clDNN/api/intel_gpu/runtime/device_info.hpp

```diff
@@ -25,6 +26,10 @@ struct gfx_version {
     uint16_t major;
     uint8_t minor;
     uint8_t revision;
+    friend bool operator < (const gfx_version& l, const gfx_version& r)  {
```

Comment:
Are these changes related to PR?

Reply:
to sort by the gfx_version in the [std::map](https://github.com/openvinotoolkit/openvino/pull/7883/files#diff-cbdb092fdcbc1aaddd07cd8cf329b73c35c1834bfa35a9420872989eb12c0e20R732)